corehq/motech/repeaters/models.py

```diff
@@ -298,6 +299,28 @@ class Meta:
     def repeater(self):
         return Repeater.get(self.repeater_id)
 
+    def to_json(self):
+        repeater_dict = self.__dict__.copy()
+        options = repeater_dict.pop('options', None)
+
+        # Populating OptionValue attrs
+        for attr_tuple in inspect.getmembers(self.__class__):
```

Comment:
`inspect.getmembers(self.__class__)` might be an expensive call. I forget the performance characteristics, but know that some things in `inspect` should be used carefully. Do you know if this could ever cause performance issues?